modules/howtos/examples/transactions-example.php

```diff
@@ -255,92 +284,110 @@ function (TransactionAttemptContext $ctx) use ($hotelChain, $country) {
+      );
+    }
+  );
+  // end::queryRYOW[]
 }
 
-function queryRYOW($cluster) {
-  // tag::queryRYOW[]
+function queryKvMix($cluster, $collection)
+{
+  echo "\nRunning: queryKvMix example\n";
+  // tag::queryKvMix[]
   $cluster->transactions()->run(
-    function (TransactionAttemptContext $ctx) {
-      $qr = $ctx->query("UPDATE `travel-sample`.inventory.hotel SET price = 99.00 WHERE name LIKE \"Marriott%\"");
-      if ($qr->metaData()->metrics()["mutationCount"] != 2) {
-        // throw new \Exception("Mutation count not the expected amount.");
-        echo "WARN: Mutation count not the expected amount.\n";
-      }
+    function (TransactionAttemptContext $ctx) use ($collection) {
+      // Key-Value insert
+      $ctx->insert($collection, "doc-greeting", ["greeting" => "Hello World"]); // <1>
+
+      // Query SELECT 
+      $selectQuery = "SELECT greeting FROM `travel-sample`.inventory.airline WHERE META().id = 'doc-greeting'";
+      $ctx->query($selectQuery); // <2>
     }
   );
-// end::queryRYOW[]
+  // end::queryKvMix[]
 }
 
-function queryOptions($cluster) {
+function queryOptions($cluster)
+{
+  echo "\nRunning: queryOptions example\n";
   // tag::queryOptions[]
-
   $cluster->transactions()->run(
     function (TransactionAttemptContext $ctx) {
       $txQo = TransactionQueryOptions::build()
         ->readonly(false)
         ->positionalParameters(["key", "value"]);
-        
+
       $ctx->query(
         "UPSERT INTO `travel-sample`.inventory.airline VALUES ('docId', {\$1:\$2})",
-        $txQo);
-  });
+        $txQo
+      );
+    }
+  );
   // end::queryOptions[]
 }
 
-function querySingle($cluster) {
-  // tag::querySingle[]
-  try {
-    $cluster->transactions()->run(
-      function (TransactionAttemptContext $ctx) {
-        $bulkLoadStatement = "..."; // a bulk-loading N1QL statement not provided here
-
-        $ctx->query($bulkLoadStatement);
-    });
-  }
-  catch (\Couchbase\Exception\TransactionOperationFailedException $e) {
-    echo "Transaction did not reach commit point\n";
-  }
-  catch (\Couchbase\Exception\TransactionException $e) { 
-    echo "Transaction possibly committed\n";
-  }
-  // end::querySingle[]
-}
+// TODO: Verify this is the correct example for a single query transaction. 
+// function querySingle($cluster)
+// {
+//   echo "\nRunning: querySingle example\n";
+//   // tag::querySingle[]
+//   try {
+//     $cluster->transactions()->run(
+//       function (TransactionAttemptContext $ctx) {
+//         $bulkLoadStatement = "..."; // a bulk-loading N1QL statement not provided here
+
+//         $ctx->query($bulkLoadStatement);
+//       }
+//     );
+//   } catch (\Couchbase\Exception\TransactionFailedException $e) {
+//     echo "Transaction did not reach commit point\n";
+//   } catch (\Couchbase\Exception\TransactionCommitAmbiguousException $e) {
+//     echo "Transaction possibly committed\n";
+//   }
+//   // end::querySingle[]
+// }
```

Comment:
@avsej  and @programmatix are **single query transactions** available for PHP?
I am unclear how we show this in PHP, from what I can see in Java (using as reference), this looks a little different?

https://docs.couchbase.com/java-sdk/current/howtos/distributed-acid-transactions-from-the-sdk.html#single-query-transactions

There seems to be a `QueryOptions.queryOptions().asTransaction()` method, but I couldn't find anything similar in PHP.